### ISSUE TITLE
Makefile, make source not working properly

### ISSUE THREAD

**Issue by siso** [boxidau/rax-autoscaler#56]
Commented lines are ignored when running `$ make source`:

```
source:
    pandoc -f markdown_github -t plain -o README.txt README.md  # IGNORED
    $(PYTHON) setup.py sdist
    rm -rf RAX_AutoScaler.egg-info  # IGNORED
```

**Comment by siso** [comment]
`$ brew install pandoc`

**Comment by sthapa-ping** [comment]
Is it not working now because you  commented pandoc in revision  3c59bb3 fix #55 ?  It working perfectly for me as I can see it created README.txt file and removed  RAX_AutoScaler.egg-info from current dir.
